drf_stripe/stripe_api/checkout.py

```diff
@@ -103,7 +103,7 @@ def _make_stripe_checkout_params(
 
     print(success_url)
```

Comment:
I forgot to remove this print statement, maybe you can remove this with your next commit too.